### ISSUE TITLE
Backlight Control

### ISSUE THREAD

**Issue by fvanroie** [Makerfabs/Project_Touch-Screen-Camera#8]
Does the board have a way to control the backlight brightness via GPIO?

I see in the schematic:
![image](https://user-images.githubusercontent.com/15969459/124366349-7649b080-dc4f-11eb-8f31-319ced732117.png)

Do I need to solder TXD to LCD_BCK on the board?

**Comment by Makerfabs** [comment]
@fvanroie Yes, as you can see from the schematic(https://github.com/Makerfabs/Project_Touch-Screen-Camera/tree/master/hardware/ ) , all the pins are connected to camera/LCD/SD card, there only TXD/RXD left.  If you want to control the backlight,  solder R31(0R,0603) resistance on the board and set TXD as GPIO1 for PWM output(You can't see the uart  debug message any more).
   Generally speaking, if you had finished your code after a lot of testing. The released firmware didn't need debug message any more, so you can set TXD pin as PWM output pin to control backlight.

**Comment by FrancisGut** [comment]
Hello, I just soldered the R31 resistance and I want to control the tft brightness in Arduino but I dont find the way to do it. Have you some comments about it, please? Thanks.

**Comment by FrancisGut** [comment]
Hello, the problem is solved. Sorry I have setted the pin 35 as TXD but is the GPIO1, so the pin to write the analog values is the 1. Thanks.

**Comment by mmccool** [comment]
This is great to know.  I wanted to use this board with openHASP but the lack of backlight control is a big problem for a home control panel.  I don't care about TX so... and I guess technically I don't even need a resistor, I can just bridge the two pads with a blob of solder.  But now I need to hack the openHASP code to understand it needs to use GPIO1 :)
